packages/react/src/primitives/Tabs/Tabs.tsx

```diff
@@ -0,0 +1,87 @@
+import React from 'react';
+import {
+  Root,
+  List,
+  Trigger as RadixTab,
+  Content as Panel,
+} from '@radix-ui/react-tabs';
```

Comment:
The main reasons for using [Radix](https://www.radix-ui.com/docs/primitives/components/tabs) is because it adheres to the Tabs WAI-ARIA design pattern and is also a small package size (3.35 kB).